cmake/partition_manager.cmake

```diff
@@ -437,8 +450,66 @@ else()
             )
 
           # There is no padding in front of the network core application.
-          math(EXPR net_app_TO_SECONDARY
-            "${PM_MCUBOOT_SECONDARY_ADDRESS} - ${net_app_addr} + ${PM_MCUBOOT_PAD_SIZE}")
+	  if(CONFIG_PM_EXTERNAL_FLASH)
+	    if(CONFIG_SOC_NRF5340_CPUAPP)
+	      set(EXTERNAL_FLASH_START_ADDRESS 0x10000000)
```

Comment:
These hard coded symbols should be prompt-less kconfig-options IMO

Reply:
Unfortunately, these values are not found in the device tree. They are currently fetched from https://infocenter.nordicsemi.com/topic/ps_nrf5340/chapters/memory/appmem.html?cp=3_0_0_4_2 and 
https://infocenter.nordicsemi.com/topic/ps_nrf52840/memory.html?cp=4_0_0_3_1_2#memorymap

IMO they should have been encoded in the DTS of the SoC but I couldn't find a suitable node.